index.html

```diff
@@ -38,91 +38,102 @@
+            countries.</p>
         </div>
-        <div class="container-card__four">
-          <img class="card-four__img1" src="./assets/img/icons/spanish.png" alt="Spanish as a topic of the conference" tabindex="0"/>
-          <h3  class="card-four__h3" tabindex="0">Spanish</h3>
-          <p class="card-four__p1" tabindex="0">Spanish is the second most spoken language in the world, with more than 400 million speakers.</p>
+        <div class="container-card__four card">
+          <img class="card-four__img1" src="./assets/img/icons/spanish.png" alt="Spanish as a topic of the conference"
+            tabindex="0" />
+          <h3 class="card-four__h3" tabindex="0">Spanish</h3>
+          <p class="card-four__p1" tabindex="0">Spanish is the second most spoken language in the world, with more than
+            400 million speakers.</p>
         </div>
-        <div class="container-card__five">
-          <img class="card-five__img1" src="./assets/img/icons/english.png" alt="Business English as a topic of the conference" tabindex="0"/>
-          <h3  class="card-five__h3" tabindex="0">Business English</h3>
-          <p class="card-five__p1" tabindex="0">Business English is a specialized form of English used in the workplace.</p>
+        <div class="container-card__five card">
+          <img class="card-five__img1" src="./assets/img/icons/english.png"
+            alt="Business English as a topic of the conference" tabindex="0" />
+          <h3 class="card-five__h3" tabindex="0">Business</h3>
+          <p class="card-five__p1" tabindex="0">Business English is a specialized form of English used in the
+            professional workplace.</p>
         </div>
       </div>
       <div class="section-two__link">
-        <a class="div-a__list" href="#" aria-label="See the full list of topics" tabindex="0">See the full list of topics</a>
+        <a class="div-a__list" href="#" aria-label="See the full list of topics" tabindex="0">See the full list of
+          topics</a>
       </div>
     </section>
     <section class="section-three">
       <h2 class="section-three__h2" tabindex="0">Featured topics</h2>
       <div class="divider"></div>
-      <hr class="section-three__hr" />
-
-
       <div class="section-three__container"></div>
-
-
       <div class="section-three__button">
         <button class="div-button__more" type="submit" aria-label="See more" tabindex="0">More</button>
       </div>
     </section>
     <section class="section-four">
       <h2 clas="section-four__h2" tabindex="0">Partner</h2>
-      <hr class="section-four__h2" />
+      <div class="divider section-four__divider"></div>
       <div class="section-four__container">
         <div class="section-four-container__card1">
-          <img src="./assets/img/partner/1.png" alt="Logo of the partner" tabindex="0"/>
+          <img src="./assets/img/partner/1.png" alt="Logo of the partner" tabindex="0" />
```

Comment:
- [ ] Kindly for the Third section, I would highly recommend following the o[riginal design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), it is great to personalize the design to make it unique ✨✨. however, due to the project requirements, we had to only personalize the content 🙏

please try to follow these instructions as suggested:
  1. The section articles should be organized in two columns of three articles each. 
  2. Kindly try to replicate the original design sizing and spacing of your elements
  3. I would also recommend decreasing the shadow of your cards to make a similar design to the given template 🙏 

**Your design**
![image](https://user-images.githubusercontent.com/102757126/225830015-d57b1e59-eb95-44ac-9b64-8167321b9599.png)

**Original design**
![image](https://user-images.githubusercontent.com/102757126/225830095-b68b6a8c-51df-48a0-b3ae-259c96cb620e.png)

Reply:
Hello. Thank you very much for your feed. I fixed that:
![image](https://user-images.githubusercontent.com/95459740/226062535-dc5786bc-cbe2-422b-bfd6-d53024b4e0b0.png)

Reply:
Thank you very much. I fixed it.
Have a good day.